amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -194,11 +195,22 @@ router.get('/:startTimestamp', async (req, res) => {
  *      responses:
  *         "200":
  *            description: Success
- *         "404":
+ *         "400":
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   const startTimestamp = req.params.startTimestamp;
+   const shiftBody = {
+      startTimestamp: parseInt(startTimestamp),
+      ...req.body
+   }
+   try {
+     const shift = await putShift(shiftBody);
```

Comment:
I don't think we need to save the result of putShift if we're not gonna use it anywhere